src/Fixer/ReturnNotation/ReturnAssignmentFixer.php

```diff
@@ -468,4 +474,58 @@ private function isOpenBraceOfMatch(Tokens $tokens, int $index): ?int
 
         return $tokens[$index]->isGivenKind(T_MATCH) ? $index : null;
     }
+
+    private function isUsedInCatchOrFinally(Tokens $tokens, int $returnVarIndex, int $functionOpenIndex, int $functionCloseIndex): bool
+    {
+        // Find try
+        $tryIndex = $tokens->getPrevTokenOfKind($returnVarIndex, [[T_TRY]]);
+        if (null === $tryIndex || $tryIndex <= $functionOpenIndex || $tryIndex >= $functionCloseIndex) {
```

Comment:
```suggestion
        if (null === $tryIndex || $tryIndex <= $functionOpenIndex) {
```
The last condition is not possible, right? `$tryIndex` is smaller than `$functionCloseIndex` when we enter the `isUsedInCatchOrFinally` function and with `getPrevTokenOfKind` value of `$tryIndex` can be only smaller.